src/passes/OptimizeInstructions.cpp

```diff
@@ -3507,20 +3507,6 @@ struct OptimizeInstructions
         return curr;
       }
     }
-    {
-      double value;
-      if (matches(curr, binary(Sub, any(), fval(&value))) && value == 0.0) {
-        // x - (-0.0)   ==>   x + 0.0
-        if (std::signbit(value)) {
-          curr->op = Abstract::getBinary(type, Add);
-          right->value = right->value.neg();
-          return curr;
-        } else if (fastMath) {
```

Comment:
Don't we still want the fast-math part?

Reply:
This redundant rule. FastMath part already handled here:
https://github.com/WebAssembly/binaryen/blob/ca6a212b5a687fd5d53d222189423915f7fb1778/src/passes/OptimizeInstructions.cpp#L3526

Reply:
I see, thanks!